src/app/components/common/node-editor/custom-node/custom-node.component.html

```diff
@@ -172,8 +172,8 @@
     </div>
   </ng-container>
   <ng-template #addServiceTemplate>
-    <div class="add-service">
-      <span class="has-tooltip-bottom has-tooltip-arrow tooltip is-hovered"
+    <div [ngClass]="{'disabled-node': !rolesService.hasEditPermissions()}" [ngClass]="from == 'north' ? 'add-task' : 'add-service'">
```

Comment:
Using `ngClass` directive twice on same element actually overrides the first occurrence. To apply multiple classes with `ngClass` we could simply use comma separated syntax.
`[ngClass]="{'disabled-node': !rolesService.hasEditPermissions(), 'add-task': from=='north', 'add-service': from!='north'}"`